ext/node/package_json.rs

```diff
@@ -0,0 +1,158 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+
+use super::DenoDirNpmResolver;
+use deno_core::anyhow;
+use deno_core::anyhow::bail;
+use deno_core::error::AnyError;
+use deno_core::serde_json;
+use deno_core::serde_json::Map;
+use deno_core::serde_json::Value;
+use serde::Serialize;
+use std::io::ErrorKind;
+use std::path::PathBuf;
+
+#[derive(Clone, Debug, Serialize)]
+pub struct PackageJson {
```

Comment:
This is very similar to `PackageConfig` in `cli/compat/esm_resolver.rs`. Either combine these two structs or add a TODO.